modules/ref-cpu-metrics.adoc

```diff
@@ -0,0 +1,49 @@
+:_module-type: REFERENCE
+
+[id='ref-cpu-metrics_{context}']
+= CPU metrics
+
+[role='_abstract']
+You can track metrics on your CPU to ensure the health of the hardware.
+
+CPU utilization::
+--
+Tracks CPU usage to identify workloads that are CPU-bound.
+
+_Query_
+[source,bash]
+----
+sum(rate(container_cpu_usage_seconds_total{namespace="$namespace", pod=~"granite.*"}[5m])) by (namespace)
```

Comment:
Please replace all occurrences of `pod=~"granite.*"` with `pod=~"$model_name.*"`